### ISSUE TITLE
[Feature request]: Make Menu responsive with Flex-Wrap

### ISSUE THREAD

**Issue by jonnyfux** [sibiraj-s/ngx-editor#383]
### Proposal

Hi, 

we just encounter the problem if you have many items in the menu bar it will eventually get cut off because it doesn't wrap in a second row. This would be normal behavior that for example google docs uses. 

I suppose adding flex-wrap to the mehurbar, so it would like this 

```
.NgxEditor__MenuBar {
  display: flex;
  padding: 3px;
  cursor: default;
  background-color: #fff;
  **flex-wrap: wrap;**
}
```

Here are few screenshots of the behavior (don't mind our custom styling)
![image](https://user-images.githubusercontent.com/58081454/155528430-945bda41-5e8d-4cbb-8999-206c1e4f942b.png)

![image](https://user-images.githubusercontent.com/58081454/155528532-fab98662-426a-49dc-bc88-f9eb069d6f1e.png)

### Willing to submit a PR?

Yes

**Comment by sibiraj-s** [comment]
Yes Please, Thank you @jonnyfux

**Comment by sibiraj-s** [comment]
Fixed in v12.2.2
